.travis.yml

```diff
@@ -1,4 +1,5 @@
 sudo: false
+dist: bionic
```

Comment:
Xenial had an older version of unixodbc that didn't support static linking properly.